authors/apps/profiles/test_follow.py

```diff
@@ -0,0 +1,100 @@
+from django.contrib.auth import get_user_model
+from rest_framework.test import APITestCase
+from rest_framework.reverse import reverse
+from rest_framework import status
+from .models import Profile
+
+
+from authors.apps.authentication.models import UserManager
+
+
+class ProfileTest(APITestCase):
+    """
+    User authentication test cases
+    """
+    def setUp(self):
+        """
+        Method for setting up user
+        """
+        self.user =  { 
+            "user":{
+                    'username': 'janeDoe', 
+                    'email': 'jane@doe.com', 
+                    'password': 'janedoe123', 
+            }
+        }
+
+        self.user_2 =  { 
+            "user":{
+                    'username': 'jane1Doe', 
```

Comment:
ditto

Reply:
@FawazFarid, just made the changes and made indentation to 4 spaces